### DISCUSSION TITLE
How to use Luxon without import?

### DISCUSSION THREAD

**Discussion by GregJohnStewart** [moment/luxon#1453] [Q&A] [answered]
I have a more 'basic' setup for including js files, and am having trouble bringing Luxon in. For sure a result of my patchwork understanding of JS.

I am bringing in the scripts like:

```html
<script src="/lib/luxon/3.3.0/luxon.min.js"></script>
<script src="/res/js/timeHelpers.js"></script>
```

TimeHelpers: (errs outlined in comments)
```javascript
import { DateTime, Settings, Info, Duration } from "../../lib/luxon/3.3.0/luxon.min"; //Fails as TimeHelpers isn't a module

const TimeHelpers = {

	/**
	 * Get the current timestamp used for `datetime-local` inputs.
	 */
	getNowTs(){
		return DateTime.now().toJSDate(); //can't find 'DateTime' without import
	}
}
```

**Comment by diesieben07** [comment, marked as answer]
If you're just working in the browser and load Luxon directly, its exports are visible through the `luxon` global, as shown [here in the documentation](https://moment.github.io/luxon/#/install?id=basic-browser-setup).
Please let me know if this helps!

**Reply by GregJohnStewart** [reply]
Somehow missed that, thanks!
